lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -453,7 +453,7 @@ describe('GatherRunner', function() {
       ],
     };
 
-    return GatherRunner.afterPass({url, driver, passConfig}, {TestGatherer: []}).then(passData => {
```

Comment:
could you add a quick test for the `skipNavigation` use case? it might make sense to do it at the highest `run` level to get most bang for buck

Reply:
Yeah I was going to ask where the test for this could go. I was having trouble figuring out how fake the fake-driver was.